truss/templates/server/common/truss_server.py

```diff
@@ -255,6 +256,20 @@ def create_application(self):
             },
         )
 
+        async def dispatch(request, call_next):
+            request_id = request.headers.get("x-baseten-request-id", uuid.uuid4())
+            with loguru_logger.contextualize(request_id=request_id):
+                try:
+                    return await call_next(request)
+
+                except Exception as ex:
```

Comment:
why do we need to add this additional exception handling? Errors are already handled in truss, we also return an error message of "Internal Server Error". 

See the model_wrapper code for how some of this is handled: https://github.com/basetenlabs/truss/blob/main/truss/templates/server/model_wrapper.py#L370